keras_nlp/layers/transformer_decoder_test.py

```diff
@@ -56,6 +70,19 @@ def test_valid_call_with_mask(self):
             use_causal_mask=True,
         )
 
+    def test_valid_call_without_encoder_with_mask(self):
```

Comment:
We can delete this test case because it is covered by `test_valid_call_with_mask`